src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -270,30 +268,6 @@ private BufferedWriter makeWriter() throws FileNotFoundException {
         );
     }
 
-    /**
-<<<<<<< HEAD
```

Comment:
this old merge marker is nice

Reply:
yeah.....we need a bot to clean that up....